UI/src/elements/lsmb-text.js

```diff
@@ -0,0 +1,78 @@
+
+const registry = require("dijit/registry");
+const dojoTextBox = require("dijit/form/TextBox");
+
+class LsmbText extends HTMLElement {
+    dojoWidget = null;
+
+    constructor() {
+        super();
+    }
+    adoptedCallback() {
+        if (this.dojoWidget
+            && this.dojoWidget.resize) {
+            this.dojoWidget.resize();
+        }
+    }
+    connectedCallback() {
+        if (this.dojoWidget) {
+            if (registry.byId(this.dojoWidget.id) !== this.dojoWidget) {
+                registry.add(this.dojoWidget);
+            }
+            return;
+        }
+        let root = this;
```

Comment:
We're *not* creating a shadowRoot here, because that separates the styling from the rest of the page, but we want the body's list of classes to influence the theme used to render the widgets. Instead, we simply modify the DOM below the tag directly.